INSTALL.md

```diff
@@ -322,13 +322,15 @@ developers mailing list. See <https://grass.osgeo.org/development/>
 ## (L) GRASS PROGRAMMER'S MANUAL
 
 The Programmer's manual is generated with doxygen from the source code.
-Please see the README file and the files at:
+Please see the [README](doc/development/README.md) file and the files at:
```

Comment:
We can solve this in another PR, but this README needs to be changed, the current content there doesn't seem very relevant. Maybe the manual building instruction could go right here.

Reply:
Ok, will do in a different PR.